### PR TITLE
docs: Finished the API apps page based on discussion with Benjamin

### PR DESCRIPTION
**Pull request by yomarion**
## Link to Jira

https://requestnetwork.atlassian.net/browse/PROT-

### CODE REVIEW

File: packages/docs/docs/guides/3-API/3-api-apps.md [hunk 1]
```diff
@@ -36,5 +39,6 @@ As an app developer, you will _not_ see your users' requests in your dashboard.
 :::
 
 
-# Get help
-TODO: Get in touch with us
\ No newline at end of file
+## Get help
+
+Do you have questions ? Or do you have troubles authenticating your app? Get in touch with us on the [Request Hub](https://requesthub.slack.com).
```
Comment (benjlevesque) [inline packages/docs/docs/guides/3-API/3-api-apps.md]: I think you need to give an invite link, this is not enough

File: packages/docs/docs/guides/3-API/3-api-apps.md [hunk 2]
```diff
@@ -3,21 +3,24 @@ title: API Apps
...
+## How it works
 
 An application built on top of Request API needs to authenticate with the OAuth2 mechanism.
 We recommend using [Auth0 SDKs](https://auth0.com/docs/libraries#sdks) to integrate it, but you are free to choose whatever you want.
 
-# Getting started
+## Getting started
 
-TODO: unlock the Apps feature for all users?? 
+Unlock this option by getting in touch with us on the [Request Hub](https://requesthub.slack.com).
```
Comment (benjlevesque) [inline packages/docs/docs/guides/3-API/3-api-apps.md]: see next comment

Comment (yomarion) [reply packages/docs/docs/guides/3-API/3-api-apps.md]:
```suggestion
Unlock this option by getting in touch with us on the [Request Hub](https://join.slack.com/t/requesthub/shared_invite/enQtMjkwNDQwMzUwMjI3LTc5NDRmN2YyMTVhZTBjNDE2MWU2YTBlYWIzYmJlYzNkMWQ5MzVmYzEzNGVmYjliNDQ4MjkyNTBiYjk4MDk3ZGE).
```

File: packages/docs/docs/guides/3-API/3-api-apps.md (hunk 1 above)
Comment (yomarion) [reply packages/docs/docs/guides/3-API/3-api-apps.md]:
```suggestion
Do you have questions ? Or do you have troubles authenticating your app? Get in touch with us on the [Request Hub](https://join.slack.com/t/requesthub/shared_invite/enQtMjkwNDQwMzUwMjI3LTc5NDRmN2YyMTVhZTBjNDE2MWU2YTBlYWIzYmJlYzNkMWQ5MzVmYzEzNGVmYjliNDQ4MjkyNTBiYjk4MDk3ZGE).
```

**Pull request merged**